Go/Go.sublime-syntax

```diff
@@ -727,6 +727,14 @@ contexts:
   match-keyword-package:
     - match: \bpackage\b
       scope: keyword.other.package.go
+      push:
+        - match: \b_\b
+          scope: variable.language.blank.go
+
+        - match: '{{ident}}'
+          scope: entity.name.package.go
```

Comment:
You can pop on these matches.

Reply:
Would we prefer the more general `entity.name.namespace.go`, wouldn't we?

Reply:
Changed `entity.name.package.go` to `entity.name.namespace.go` and also added `pop: true` for the `match: '{{ident}}'`.

Reply:
I actually meant the `_` match as well, though I'm not sure what that is for. Why declare a package when you use a dump name? Is it for single-file executables?

Reply:
Yeah, my purpose was to have easy-identifiable scope where package name starts.